src/foam/nanos/auth/CreatedByAwareDAO.js

```diff
@@ -20,6 +20,7 @@ foam.CLASS({
         }
         return this.delegate.find_(x, obj).then(function(result) {
           if ( result == null ) {
+            obj = obj.fclone();
```

Comment:
I would assume the object is already not frozen.

Reply:
well you would think, but I ended up here, because if was frozen.